pkg/controller/cronjob/cronjob_controller.go

```diff
@@ -215,12 +215,23 @@ func syncOne(sj *batchv1beta1.CronJob, js []batchv1.Job, now time.Time, jc jobCo
 		if !found && !IsJobFinished(&j) {
 			recorder.Eventf(sj, v1.EventTypeWarning, "UnexpectedJob", "Saw a job that the controller did not create or forgot: %v", j.Name)
 			// We found an unfinished job that has us as the parent, but it is not in our Active list.
-			// This could happen if we crashed right after creating the Job and before updating the status,
-			// or if our jobs list is newer than our sj status after a relist, or if someone intentionally created
-			// a job that they wanted us to adopt.
+			// This could happen if:
+			// * we crashed right after creating the Job and before updating the status
+			// * if our jobs list is newer than our sj status after a relist
+			// * if someone intentionally created a job that they wanted us to adopt
+			// * if a job was created via the instantiate subresource (will be labeled as createdByInstantiate=yes)
+
+			// if the job was created via the instantiate subresource, adopt it
+			if instantiateAnnotation, exists := j.Annotations["cronjob.kubernetes.io/instantiate"]; exists && instantiateAnnotation == "manual" {
```

Comment:
Can you make the above warning different for manual jobs, it should along the lines of controller noticed manual job...